renderer/components/Channels/ChannelCreateSummary.js

```diff
@@ -30,21 +31,23 @@ class ChannelCreateSummary extends React.Component {
   }
 
   render() {
-    const { amount, fee, speed, nodePubkey, nodeDisplayName, ...rest } = this.props
-    const speedTitleMessageKey = speed.toLowerCase()
-    const pubkey = nodePubkey.split('@')[0]
+    const {
+      amount,
+      fee,
+      speed,
+      nodePubkey,
+      nodeDisplayName,
+      lndTargetConfirmations,
+      ...rest
+    } = this.props
+    const [pubkey] = nodePubkey.split('@')
```

Comment:
Rather than doing this here, it might be better to pass in `infoSelectors.nodePubkey` via `mapStateToProps`